ui/src/app/pools/views/Pools.js

```diff
@@ -105,51 +118,66 @@ const Pools = () => {
   const resourcePools = useSelector(resourcePoolSelectors.all);
 
   return (
-    <Row>
-      <Col size={12}>
-        <div>
-          {poolsLoading && (
-            <div className="u-align--center">
-              <Loader text="Loading..." />
-            </div>
-          )}
-          {poolsLoaded && (
-            <MainTable
-              className="p-table-expanding--light"
-              defaultSortDirection="ascending"
-              headers={[
-                {
-                  content: "Name",
-                  sortKey: "name",
-                },
-                {
-                  content: "Machines",
-                  sortKey: "machines",
-                },
-                {
-                  content: "Description",
-                  sortKey: "description",
-                },
-                {
-                  content: "Actions",
-                  className: "u-align--right",
-                },
-              ]}
-              expanding={true}
-              paginate={150}
-              rows={generateRows(
-                resourcePools,
-                expandedId,
-                setExpandedId,
-                dispatch,
-                setDeleting
-              )}
-              sortable
-            />
-          )}
-        </div>
-      </Col>
-    </Row>
+    <>
+      {errorMessage ? (
+        <Row>
+          <Col size={12}>
+            <Notification
+              close={() => dispatch(resourcePoolActions.cleanup())}
+              timeout={5000}
```

Comment:
I don't think we should timeout a negative notification. I thought timeout notifications were just for informational and confirmation of user action?

Reply:
@anthonydillon a question for UX, I believe we do this in a number of places. @amylily1011 @lilyvidenova?

Reply:
We did have this guidelines when we originally created the notification component but that was a little while ago now.

Reply:
I think you're right, we have temporary timeouts for the "information" notifications that just confirm what a user has done and the "negative" notifications persist until the user navigates away. I'll remove the timeout.